DependencyInjection/Compiler/RegisterUrlInformationProviderPass.php

```diff
@@ -32,13 +32,34 @@ class RegisterUrlInformationProviderPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
+        $this->processTagsForService(
+            'cmf_seo.sitemap.document_provider',
```

Comment:
why "document" - i think this should be called "object" here, its not only about phpcr-odm.